src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue31361.cs

```diff
@@ -0,0 +1,26 @@
+#if TEST_FAILS_ON_CATALYST && TEST_FAILS_ON_WINDOWS //In Catalyst and Windows, `ScrollDown` isn't functioning correctly with Appium.
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue31361 : _IssuesUITest
+{
+    public override string Issue => "CarouselView content is scrolling vertically";
+
+    public Issue31361(TestDevice device)
+    : base(device)
+    { }
+
+    [Test]
+    [Category(UITestCategories.CarouselView)]
+    public void VerifyHorizontalCarouselViewPreventsVerticalScrolling()
```

Comment:
Can include another test where:
1. Start with horizontal carousel
2. Change to vertical at runtime
3. Verify bounce behavior updates correctly

Reply:
@jsuarezruiz, In CarouselView with vertical orientation, the bounce effect works as expected. Still, I have added a test case for this scenario.